drivers/wifi/nrf_wifi/src/wifi_util.c

```diff
@@ -915,6 +915,49 @@ static int nrf_wifi_util_trigger_rpu_recovery(const struct shell *sh,
 	k_mutex_unlock(&ctx->rpu_lock);
 	return ret;
 }
+
+static int nrf_wifi_util_rpu_recovery_info(const struct shell *sh,
+					   size_t argc,
+					   const char *argv[])
+{
+	struct nrf_wifi_fmac_dev_ctx *fmac_dev_ctx = NULL;
+	struct nrf_wifi_hal_dev_ctx *hal_dev_ctx = NULL;
+	int ret;
+
+	k_mutex_lock(&ctx->rpu_lock, K_FOREVER);
+	if (!ctx || !ctx->rpu_ctx) {
+		shell_fprintf(sh,
+			      SHELL_ERROR,
+			      "RPU context not initialized\n");
+		ret = -ENOEXEC;
+		goto unlock;
+	}
+
+	fmac_dev_ctx = ctx->rpu_ctx;
+	hal_dev_ctx = fmac_dev_ctx->hal_dev_ctx;
+
+	shell_fprintf(sh,
+		      SHELL_INFO,
+		      "wdt_irq_received: %d\n"
+		      "wdt_irq_ignored: %d\n"
+		      "last_wakeup_now_asserted_time_ms: %lu\n"
+		      "last_wakeup_now_deasserted_time_ms: %lu\n"
+		      "last_rpu_sleep_opp_time_ms: %lu\n"
```

Comment:
do these make sense without the current time?

Reply:
Adding current time now. we can add some more debug stats later.